extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/runtime/exporter/OtlpExporterTracesConfig.java

```diff
@@ -73,6 +58,37 @@ public interface OtlpExporterTracesConfig {
     @WithDefault(Protocol.HTTP_PROTOBUF)
     Optional<String> protocol();
 
+    /**
+     * Key/cert configuration in the PEM format.
+     */
+    @WithName("key-cert")
+    KeyCert keyCert();
+
+    /**
+     * Trust configuration in the PEM format.
+     */
+    @WithName("trust-cert")
+    TrustCert trustCert();
+
+    interface KeyCert {
+        /**
+         * Comma-separated list of the path to the key files (Pem format).
+         */
+        Optional<List<String>> keys();
+
+        /**
+         * Comma-separated list of the path to the certificate files (Pem format).
+         */
+        Optional<List<String>> certs();
+    }
+
+    interface TrustCert {
+        /**
+         * Comma-separated list of the trust certificate files (Pem format).
+         */
+        Optional<List<String>> certs();
+    }
```

Comment:
Curious if this works (and if makes sense?)

```suggestion
    interface KeyCert extends TrustCert {
        /**
         * Comma-separated list of the path to the key files (Pem format).
         */
        Optional<List<String>> keys();
    }

    interface TrustCert {
        /**
         * Comma-separated list of the trust certificate files (Pem format).
         */
        Optional<List<String>> certs();
    }
```

Reply:
It probably does but I wouldn't risk it because the documentation generation probably will break